kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -179,6 +179,16 @@ export default [
   {
     name: PageNames.BOOKMARKS,
     path: '/bookmarks',
+    beforeEnter: (to, from, next) => {
```

Comment:
I think we can leverage the authorized user messaging in a different way:

I know this should and will later be refactored, but while we have it, just updating the `userIsAuthorized` computed prop in `LearnIndex.vue` so that if the page is `PageName.BOOKMARKS`, then the user gets flagged as not authorized.

https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/LearnIndex.vue#L97

This feels a little bit more explicit than setting an error to 403 in the core errors, which might be interpreted by other parts of the app in another way.

Reply:
That is infinitely easier and clearer. Thanks!